### ISSUE TITLE
Have to run example with "sudo" to make remote sample download on local machine when using docker

### ISSUE THREAD

**Issue by JohnHadish** [SystemsGenetics/GEMmaker#75]
In order for "sratoolkit" to successfully download the example file on local file system using docker, I must use command:
```
sudo nextflow run main.nf -profile standard,inDocker
```
Instead of without sudo.

The error it was throwing without sudo access was this:
```
Status: Downloaded newer image for systemsgenetics/sratoolkit:2.9.2
2019-01-07T00:15:36 fastq-dump.2.9.2 err: path not found while creating directory within file system module - error with https open 'https://sra-download.ncbi.nlm.nih.gov/traces/sra8/SRR/000634/SRR649944'
2019-01-07T00:15:36 fastq-dump.2.9.2 err: item not found while constructing within virtual database module - the path 'SRR649944' cannot be opened as database or table
```

When I ran it with sudo, it ran perfectly. All other programs ran without sudo, and execution of other three samples went smoothly. This was done using a fresh copy of the most recent version. I did a fresh pull of all dockers as well.

**Comment by bentsherman** [comment]
There seems to be a discrepancy in how we're using docker on our respective machines. Normally when you install docker you have to use `sudo` to do anything with docker. However, I learned from [this article](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-16-04) that it's possible to use docker without root privileges by adding your user to some docker group or whatever. I'm assuming that's what you guys have done on your systems?

On my system I still only run docker with `sudo` which is why I have the `sudo` option enabled in the config file. Are you leaving that option enabled when you run nextflow? If not then nextflow would run the docker containers with your user rather than root, and I don't know some of the docker containers may not behave well in that case.

**Comment by biggstd** [comment]
We had docker set up as you described initially, and do not run containers in sudo. I looked back through and found that it was removed this configuration during a PR, (commit id 13f2f83540f4815142e51268e12e120316f1203d).

**Comment by bentsherman** [comment]
The `sudo` option is still there, and it's still in the latest commit. Not sure what you're referring to?

**Comment by spficklin** [comment]
Hi folks.  Do we need to fix this before v1.0 release?

**Comment by JohnHadish** [comment]
No longer relevant
